sites/example/site/one/rubygems.pan

```diff
@@ -1,28 +1,53 @@
 unique template site/one/rubygems;
 
 # Install ruby gems packages required by the frontend
-# OS Centos/SL7
+# Centos6 and Centos7 versions require different packages
+# These packages are generated by opennebula addon-installgems
 
-prefix "/software/packages";
-"{rubygem-nokogiri}"=dict();
-"{rubygem-json}"=dict();
-"{rubygem-rack}"=dict();
-"{rubygem-sinatra}"=dict();
-"{rubygem-thin}"=dict();
-"{rubygem-zendesk_api}"=dict();
-"{rubygem-sqlite3}"=dict();
-"{rubygem-sequel}"=dict();
-"{rubygem-amazon-ec2}"=dict();
-"{rubygem-uuidtools}"=dict();
-"{rubygem-curb}"=dict();
-"{rubygem-net-ldap}"=dict();
-"{rubygem-builder}"=dict();
-"{rubygem-trollop}"=dict();
-"{rubygem-treetop}"=dict();
-"{rubygem-parse-cron}"=dict();
-"{rubygem-aws-sdk}"=dict();
-"{rubygem-ox}"=dict();
-"{rubygem-mysql}"=dict();
-"{rubygem-softlayer_api}"=dict();
-"{rubygem-configparser}"=dict();
-"{rubygem-azure}"=dict();
+'/software/packages/' = if ( OS_VERSION_PARAMS['majorversion'] == '7' )
+{
+    npush(
```

Comment:
Instead of this nested `npush(escape(dict(` stuff, it would be neater to just use `pkg_repl()`, eg:
```pan
'/software/packages/' = if ( OS_VERSION_PARAMS['majorversion'] == '7' )
{
    pkg_repl('rubygem-nokogiri');
    pkg_repl('rubygem-json');
```
etc.